src/Controller/RecurlySubscriptionListController.php

```diff
@@ -23,6 +25,32 @@
 class RecurlySubscriptionListController extends ControllerBase {
 
   /**
+   * The formatting service.
+   *
+   * @var \Drupal\recurly\RecurlyFormatManager
+   */
+  protected $formatter;
```

Comment:
@markdorison I could see that being the case if this were a service shared amongst modules, but I'm assuming it's contained to Recurly, no? It's not a big deal to change it, I suppose it makes the code more obvious.

### Currently
`$this->formatter->formatCurrency();`

### Proposed
`$this->recurly_formatter->formatCurrency();`